explore/src/main/scala/explore/tabs/TargetTabContents.scala

```diff
@@ -648,11 +677,11 @@ object TargetTabContents extends TwoPanels:
       }
       .useToastRef
       // Selected targets on the summary table
-      .useStateView(List.empty[Target.Id])
```

Comment:
Con we still multi select?

Reply:
> Con we still multi select?

Yes, but the editor is not shown in that case.